tests/nest_tests/nest_custom_templates_test.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# nest_custom_templates_test.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+import os
+import unittest
+
+import nest
+
+from pynestml.frontend.pynestml_frontend import to_nest
+from pynestml.utils.model_installer import install_nest
+
+
+class NestCustomTemplatesTest(unittest.TestCase):
```

Comment:
Is this test not redundant with ``test_code_generation_with_code_opts()`` in ``tests/pynestml_frontend_test.py``?

Reply:
This test has additional commands for installing it in NEST as an extension module. Otherwise, it is the same. Maybe I could remove this`tests/pynestml_frontend_test.py`?

Reply:
OK, cheers!